docs/arguments.md

```diff
@@ -230,6 +230,19 @@ __Do not__ Monitor and update containers that have `com.centurylinklabs.watchtow
 no `--label-enable` argument is passed. Note that only one or the other (targeting by enable label) can be 
 used at the same time to target containers.
 
+## Filter by disabling specific container names
+Monitor and update containers whose names are not in a given set of names.
+
+This can be used to exclude specific containers, when setting labels is not an option.
+The listed containers will be excluded even if they have the enable filter set to true.
+
+```text
+            Argument: --disable-containers, -x
+Environment Variable: WATCHTOWER_DISABLE_CONTAINERS
+                Type: Comma-separated string list
```

Comment:
This can also be space-separated. Perhaps that should be noted as well (if it was intended).

Reply:
It was kind of intended to be consistent with viper's standard, but on a second thought there's no reason to, updating regex instead.

Reply:
Done - do you think it'd make sense to separately just make envStringSlice always accept commas?